application/templates/cms/create_table_2.html

```diff
@@ -213,6 +212,12 @@ <h2 class="heading-medium">Data</h2>
                         <input id="table_column_5_name" class="column_option_picker_name margin-top-small">
                     </div>
                 </form>
```

Comment:
The form should close after then new `Options` input here.

Reply:
left until reply to above